lib/modules.nix

```diff
@@ -855,11 +855,7 @@ let
       else {};
 
   in {
-    inherit defsFinal mergedValue isDefined optionalValue;
-    defsFinal' = {
-      inherit highestPrio;
-      values = defsFinal;
-    };
```

Comment:
Same for this commit, it changes the interface of the effectively public function `mergeDefinitions`. A separate `mergeDefinitionsInternal` or so would be needed.